quickstarts/microsoft.batch/batchaccount-with-storage/main.bicep

```diff
@@ -0,0 +1,49 @@
+@description('Batch Account Name')
+param batchAccountName string = '${toLower(uniqueString(resourceGroup().id))}batch'
+
+@description('Storage Account type')
+@allowed([
```

Comment:
I don't think batch supports all of these account types

Reply:
Preetha Bannu Kannappan (Azure Batch PM) will find out the supported SKUs.  Until then, revert the SKU list.